gst/nnstreamer/tensor_query/tensor_query_serversink.c

```diff
@@ -265,6 +279,29 @@ gst_tensor_query_serversink_stop (GstBaseSink * bsink)
   return TRUE;
 }
 
+/**
+ * @brief An implementation of the set_caps vmethod in GstBaseSinkClass
+ */
+static gboolean
+gst_tensor_query_serversink_set_caps (GstBaseSink * bsink, GstCaps * caps)
+{
+  GstTensorQueryServerSink *sink = GST_TENSOR_QUERY_SERVERSINK (bsink);
+
+  if (!sink->caps) {
```

Comment:
Q: about sink->caps, is it necessary?
IMO, we may get the caps string from caps and replace it when called set-caps callback from basesink.

Reply:
Right. Removed.
Thanks.